Fixing an apparent typo in pascal solution 3's Dockerfile

sonicrules1234 opened the pull request:
## Description
<!--
Add your description here.
-->
Now it actually runs the benchmark.
## Contributing requirements
<!--
Make sure your PR conforms to the requirements set out in CONTRIBUTING.md:
-->

<!--
When ticking below boxes, please don't leave spaces between the 'x' and the square brackets, as that breaks the checkbox rendering in the PRs.
Right: [x]
Wrong: [x ]
-->
* [x] I read the contribution guidelines in CONTRIBUTING.md.
* [x] I placed my solution in the correct solution folder.
* [x] I added a README.md with the right badge(s).
* [x] I added a Dockerfile that builds and runs my solution.
* [x] I selected `drag-race` as the target branch.
* [x] All code herein is licensed compatible with BSD-3.

rbergen commented:
I think you are correct in identifying an issue with this solution's Dockerfile, but it's of a slightly different nature. The target directory should not be `/` but `/opt/app/`. Your change copies /opt/app/PrimePas from the build image to /opt/app/run.sh; this makes that only one of the two intended implementations are executed. If both PrimePas and run.sh are copied to /opt/app/ then both the single-threaded and multi-threaded implementations are executed.

rbergen commented:
(Also, on a sidenote, I think that after this PR is merged, you may want to rebase your branch to the current state of this one. As it stands, the commit list grows with every PR you open.)

sonicrules1234 commented:
Ok, I fixed pascal solution 3's Dockerfile.  I also fixed csharp solutions 2 and 3's dockerfiles by adding a ./ in front of their entry points because it was giving an error that they weren't in PATH.
